src/skin/options-layout.css

```diff
@@ -280,6 +280,54 @@ header {
     margin-inline-start: 25px;
 }
 
+#tip-container {
```

Comment:
More trailing space will be good I think. Between the tip box and the "Privacy Badger is disabled on the following sites..." text